SpongeAPI

```diff
@@ -1 +1 @@
-Subproject commit a0743842b5944fe28eb351f4cb7adb4f4e403ecc
+Subproject commit 715883a3be77319f375c65c0af2a0445c863b159
```

Comment:
https://github.com/SpongePowered/SpongeAPI/pull/824
Changed return type of ImmutableBannerData.baseColor() to use DyeColor instead of Color.